akka-operator/templates/040-deployment.yaml

```diff
@@ -90,7 +90,11 @@ spec:
         - name: {{ .Chart.Name }}
           securityContext:
             {{- toYaml .Values.securityContext | nindent 12 }}
+          {{- if and (eq (toString .Values.provider.name) "gcp") (eq (toString .Values.gcp.deployer) "false") }}
```

Comment:
if you are deploying from command line (plain Helm commands) and you are targeting GCP the default image is the one published to the marketplace